discord/utils.py

```diff
@@ -119,7 +119,10 @@ def __repr__(self) -> str:
 # field(default=MISSING) produces the same error, but passing a lambda to
 # default_factory produces the same behavior as default=MISSING and does not raise an
 # error.
-MissingField = field(default_factory=lambda: MISSING)
+
+
+def missing_field_factory() -> field:
+    return field(default_factory=lambda: MISSING)
```

Comment:
maybe I'm stupid but how is this different

Reply:
Because the other one was referencing the same object every time, whereas here it creates a new object. I have no knowledge of the python interpreter internals that make this work, but yeah